assets/questions/aad.js

```diff
@@ -0,0 +1,309 @@
+// Copyright 2017 The TIE Author, s. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Alien language alphabet.
+ */
+
+globalData.questions['aad'] = {  // eslint-disable-line dot-notation
+  title: 'Alien Language Alphabet',
+  starterCode: {
+    python:
+`def find_dict(words):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`
+class AuxiliaryCode(object):
+    @classmethod
+    def wrongInNoSolution(cls, words):
+      e = {}
+      deg = {}
+  
+      for str in words:
+          for ch in str:
+              ch = ch.lower()
+              if(not e.has_key(ch)):
+                  e[ch] = []
+                  deg[ch] = 0
+  
+      for i in range(len(words) - 1):
+          for j in range(min(len(words[i]), len(words[i + 1]))):
+              be = words[i][j].lower()
+              to = words[i + 1][j].lower()
+              if(be != to):
+                  e[be].append(to)
+                  deg[to] = deg[to] + 1
+                  break
+  
+      ans = ""
+  
+      while True :
+          flag = False
+          for key in deg:
+              if deg[key] == 0 :
```

Comment:
What should I write if I wanna check if a Variable is zero? 
is "if 0 == deg[key]: " right?